blogposts/liveblogs/gophercon/code-intelligence-with-lsif.md

```diff
@@ -11,6 +11,15 @@ published: true
 description: 'Code intelligence with LSIF'
 ---
 
+(setq markdown-xhtml-header-content
```

Comment:
Instead of this, you should be able to just wrap the iframe in `<div class="text-center">`.

Reply:
Thank you @attfarhan ! Updated it accordingly